proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,471 @@
+a year are considered stale. Without further interaction, they will be closed.
+
+### Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer publicly
+labels the issue ready for contribution. This stage indicates any user is
+welcome to initiate a contribution via a GitHub pull request. All contribution
+ready issues are listed publicly for the community to discover.
+
+### Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The maintainers will respond to let the contributor
+know to expect a preliminary review within 5 business days to confirm:
+
+1. It is implemented in the manner described in the issue.
+1. It meets the criteria specified in the contribution guide. See the
+   [appendix](#preliminary-review-criteria) for details.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to the contributing guide documentation or ask the
+maintainers to clarify. Once the contributor makes the required changes, the
+maintainers will follow up within 5 business days.
+
+If the contributor is no longer actively engaged by responding to comments from
+maintainers or making code changes for more than 30 business days, the
+contribution is stale. If not, the pull request will be closed and the
+corresponding issue will be marked as ready for contribution.
+
+### Ready for review stage
+
+Once the preliminary review criteria from the implementation stage have been
+met, the tracking GitHub issue is labeled as ready for review and added to a
+queue for maintainer review. The maintainers will select issues for full review
+from this queue. The issues will be prioritized for review publicly based on the
+following criteria in descending order of importance:
+
+1. Bugs, documentation changes, and feature requests will be prioritized in that
+   order.
+1. The time available to the maintainers will be considered against the
+   complexity of the contribution.
+1. Older contributions will be selected before more recent ones.
+
+A maintainer must select the issue from the queue with the current highest
+priority. The prioritization of issues in the queue is dynamic, reviewed on a
+regular cadence, and is ultimately decided based on the maintainers' discretion.
+Having a pull request selected for review does not guarantee how long it will be
+until it is merged.
+
+### Review stage
+
+Selecting a pull request for review will move the tracking GitHub issue to the
+under review stage. The maintainers will limit the number of active reviews to
+[TBD]. This criteria for accepting a pull request includes:
```

Comment:
I also think that these bullet points should be noted of examples of things that are considered during the full review but it is not exhaustive